_posts/2017/enterdate-building-multiple-repository-pr-test.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Build a multiple repository pull request test"
+date: "2017-01-05 10:00:00"
+image: blog-header/challenge-cube.jpg
+categories: tech-stories
+authors: ["Florian"]
+---
```

Comment:
Please write one sentence per line (better readability in md and better proof-reading in GitHub) ;-)